include/natalie/process_module.hpp

```diff
@@ -11,10 +13,168 @@ namespace Natalie {
 
 class ProcessModule : public Object {
 public:
+    static void build_constants(Env *env, ModuleObject *klass) {
+#ifdef PRIO_PGRP
+        klass->const_set("PRIO_PGRP"_s, Value::integer(PRIO_PGRP));
+#endif
+#ifdef PRIO_PROCESS
+        klass->const_set("PRIO_PROCESS"_s, Value::integer(PRIO_PROCESS));
+#endif
+#ifdef PRIO_USER
+        klass->const_set("PRIO_USER"_s, Value::integer(PRIO_USER));
+#endif
+
+#ifdef RLIMIT_AS
+        klass->const_set("RLIMIT_AS"_s, Value::integer(RLIMIT_AS));
+#endif
+#ifdef RLIMIT_CORE
+        klass->const_set("RLIMIT_CORE"_s, Value::integer(RLIMIT_CORE));
+#endif
+#ifdef RLIMIT_CPU
+        klass->const_set("RLIMIT_CPU"_s, Value::integer(RLIMIT_CPU));
+#endif
+#ifdef RLIMIT_DATA
+        klass->const_set("RLIMIT_DATA"_s, Value::integer(RLIMIT_DATA));
+#endif
+#ifdef RLIMIT_FSIZE
+        klass->const_set("RLIMIT_FSIZE"_s, Value::integer(RLIMIT_FSIZE));
+#endif
+#ifdef RLIMIT_MEMLOCK
+        klass->const_set("RLIMIT_MEMLOCK"_s, Value::integer(RLIMIT_MEMLOCK));
+#endif
+#ifdef RLIMIT_MSGQUEUE
+        klass->const_set("RLIMIT_MSGQUEUE"_s, Value::integer(RLIMIT_MSGQUEUE));
+#endif
+#ifdef RLIMIT_NICE
+        klass->const_set("RLIMIT_NICE"_s, Value::integer(RLIMIT_NICE));
+#endif
+#ifdef RLIMIT_NOFILE
+        klass->const_set("RLIMIT_NOFILE"_s, Value::integer(RLIMIT_NOFILE));
+#endif
+#ifdef RLIMIT_NPROC
+        klass->const_set("RLIMIT_NPROC"_s, Value::integer(RLIMIT_NPROC));
+#endif
+#ifdef RLIMIT_NPTS
+        klass->const_set("RLIMIT_NPTS"_s, Value::integer(RLIMIT_NPTS));
+#endif
+#ifdef RLIMIT_RSS
+        klass->const_set("RLIMIT_RSS"_s, Value::integer(RLIMIT_RSS));
+#endif
+#ifdef RLIMIT_RTPRIO
+        klass->const_set("RLIMIT_RTPRIO"_s, Value::integer(RLIMIT_RTPRIO));
+#endif
+#ifdef RLIMIT_RTTIME
+        klass->const_set("RLIMIT_RTTIME"_s, Value::integer(RLIMIT_RTTIME));
+#endif
+#ifdef RLIMIT_SBSIZE
+        klass->const_set("RLIMIT_SBSIZE"_s, Value::integer(RLIMIT_SBSIZE));
+#endif
+#ifdef RLIMIT_SIGPENDING
+        klass->const_set("RLIMIT_SIGPENDING"_s, Value::integer(RLIMIT_SIGPENDING));
+#endif
+#ifdef RLIMIT_STACK
+        klass->const_set("RLIMIT_STACK"_s, Value::integer(RLIMIT_STACK));
+#endif
+#ifdef RLIM_INFINITY
+        klass->const_set("RLIM_INFINITY"_s, IntegerObject::create(Integer(BigInt(RLIM_INFINITY))));
+#endif
+#ifdef RLIM_SAVED_CUR
+        klass->const_set("RLIM_SAVED_CUR"_s, IntegerObject::create(Integer(BigInt(RLIM_SAVED_CUR))));
+#endif
+#ifdef RLIM_SAVED_MAX
+        klass->const_set("RLIM_SAVED_MAX"_s, IntegerObject::create(Integer(BigInt(RLIM_SAVED_MAX))));
+#endif
+#ifdef WNOHANG
+        klass->const_set("WNOHANG"_s, Value::integer(WNOHANG));
+#endif
+#ifdef WUNTRACED
+        klass->const_set("WUNTRACED"_s, Value::integer(WUNTRACED));
+#endif
```

Comment:
What do you think about defining these in a new `src/process.rb` kind of like we do with [lib/socket/constants.rb](https://github.com/natalie-lang/natalie/blob/master/lib/socket/constants.rb)? Just seems cleaner.

Reply:
Sounds like a good idea, agree that it'd be much cleaner - but i'm not sure how it should be loaded.
The stuff in /lib/ all seems to be `require`d by ruby code as needed, whereas `Process` should loaded automatically as part of ruby core.  Is there a place where process.rb should be called?